menas/ui/components/mappingTable/addMappingTable.fragment.xml

```diff
@@ -14,50 +14,45 @@
-								<StandardTreeItem title="{name}" type="active"></StandardTreeItem>
-							</Tree>
-						</ScrollContainer>
-						<Label text="Selected Path"></Label>
-						<Text text="{/newMappingTable/hdfsPath}"></Text>
-					</form:content>
-				</form:SimpleForm>
-			</VBox>
-		</content>
-		<buttons>
-			<Button id="newMappingTableAddButton" text="Save" icon="sap-icon://save"></Button>
-			<Button id="newMappingTableCancelButton" text="Cancel" icon="sap-icon://cancel"></Button>
-		</buttons>
-	</Dialog>
+    xmlns:table="sap.ui.table" xmlns:hdfs="components.hdfs">
+    <Dialog id="addMappingTableDialog" title="{/newMappingTable/title} Mapping Table" resizable="true" draggable="true" class="minW40rem">
+        <content>
+            <VBox class="sapUiSmallMargin">
+                <form:SimpleForm adjustLabelSpan="true" editable="true">
+                    <form:content>
+                        <Label text="Mapping Table Name"></Label>
+                        <Input id="newMappingTableName" enabled="{path:'/newMappingTable/isEdit', formatter:'Formatters.not'}" type="text"
+                            placeholder="Enter mapping table name" width="100%" value="{/newMappingTable/name}"></Input>
+                        <Label text="Mapping Table Description"></Label>
+                        <Input id="newMappingTableDescription" type="text" placeholder="Enter mapping table description" width="100%"
+                            value="{/newMappingTable/description}"></Input>
+                        <Label text="Schema Name"></Label>
+                        <Select id="newMappingTableSchemaNameSelect" showSecondaryValues="true" items="{/schemas}" forceSelection="false"
+                            selectedKey="{/newMappingTable/schemaName}">
+                            <core:ListItem key="{_id}" text="{_id}" additionalText="Latest version: {latestVersion}"></core:ListItem>
+                        </Select>
+                        <Label text="Schema Version"></Label>
+                        <FlexBox justifyContent="SpaceBetween" class="menasAddMappingFlex">
+                            <Select id="newMappingTableSchemaVersionSelect" class="menasAddMappingSelect" items="{/currentSchemaVersions}"
+                                showSecondaryValues="true" forceSelection="false" width="auto" selectedKey="{/newMappingTable/schemaVersion}">
+                                <core:ListItem key="{version}" text="{version}"
+                                    additionalText="updated: {path: 'lastUpdated', formatter: 'Formatters.stringDateShortFormatter'}"></core:ListItem>
+                            </Select>
+                            <Button icon="sap-icon://action">
+                            </Button>
+                        </FlexBox>
+                        <Label text="HDFS Path"></Label>
+                        <hdfs:HDFSBrowser id="addMtHDFSBrowser" height="300px" width="100%" busyControl="addMappingTableDialog"
```

Comment:
We need this in `addDataset.frament.xml`, too.

Reply:
on it...